deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/resource/ApiVersionResponseFilter.java

```diff
@@ -0,0 +1,44 @@
+package org.deegree.services.oaf.resource;
+
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.ext.Provider;
+
+import org.deegree.services.oaf.workspace.DeegreeWorkspaceInitializer;
+
+
+/**
+ * Class to add API version to all the response headers in the below format
+ * API-Version: 1.0.2
+ * 
+ * @author Kapil Agnihotri
+ *
+ */
+@Provider
+public class ApiVersionResponseFilter implements ContainerResponseFilter {
+
+	// Whether to add API version to the header or not
+	private boolean addApiVersionToHeader = true;
+
+	// Constant for API-Version
+	private static final String API_VERSION_CONSTANT = "API-Version";
+
+	// API version information based on
+	// https://github.com/opengeospatial/ogcapi-features#using-the-standard
+	private static final String API_VERSION = "1.0.1";
```

Comment:
Please use the existing constant in `OpenApiCreator` instead, and change that to `1.0.1`.